lib/utils/open-url-prompt.js

```diff
@@ -47,14 +47,10 @@ const promptOpen = async (npm, url, title, prompt, emitter) => {
     if (emitter && emitter.addListener) {
       emitter.addListener('abort', () => {
         rl.close()
-
-        // clear the prompt line
-        output.standard('')
-
```

Comment:
This is no longer needed because we clear the prompt line in `display.js`. This is good because currently we wouldn't clear the input in cases besides the emitter's abort event.